src/Stratis.Bitcoin/P2P/PeerConnector.cs

```diff
@@ -230,6 +232,13 @@ public void StartConnectAsync()
                 return;
             }
 
+            // Connect if local, ip range filtering disabled or ip range filtering enabled and peer in a different group. 
+            if (!peerAddress.Endpoint.Address.IsLocal() && this.ConnectionSettings.IpRangeFiltering && this.PeerIsPartOfExistingGroup(peerAddress))
```

Comment:
I believe here it should not be `IsLocal`, but `IsRoutable(false)`

Reply:
The reason for that is that IP addresses such as 192.168.0.1 are local network addresses and IsLocal only considers localhost - 127.0.0.x

Reply:
Made this substitution.  Logic therefore ignores filtering behaviour for routable IPs.